### PR TITLE
Migrate Call Record API documentation from beta to v1.0

### PR DESCRIPTION
**Pull request by GageAmes**
Migrate the Communications Call Record API documentation from beta to v1.0. The beta documentation was added in PR #6855

### CODE REVIEW

File: api-reference/v1.0/api/callrecords-callrecord-get.md [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+
+## Examples
+
+### Example 1: Get basic details
+
+#### Request
+
+The following is an example of the request to get the basic details from a [callRecord](../resources/callrecords-callrecord.md).
+
+# [HTTP](#tab/http)
```
Comment (Lauragra) [inline api-reference/v1.0/api/callrecords-callrecord-get.md:60]: Please remove the H1 snippet sections. When we merge into master, the automation that runs on that branch will create the v1.0 snippets and insert the links in the topic. Currently this is causing the PR to fail validation.

Comment (GageAmes) [reply api-reference/v1.0/api/callrecords-callrecord-get.md:60]: Will do!  Thanks for the details.

**Comment by GageAmes** [comment]
I was informed by Angela Chu-Hatoun that we need to update the way we're documenting enumerations.  I'll make a separate pull request for those changes in beta and then update this one as well.

**Comment by GageAmes** [comment]
The pull request to update the way enum values are documented in a subnamespace is #8069.  Once that pull request is validated, I'll reflect the same changes in the v1.0 files here.

**Comment by GageAmes** [comment]
No new content was added nor technical changes made between the beta documentation and this v1.0 documentation, so requesting a content review.

File: concepts/webhooks.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Using the Microsoft Graph API, an app can subscribe to changes on the following
 - Content within the hierarchy of _any folder_ [driveItem][] on a user's personal OneDrive
 - Content within the hierarchy of the _root folder_ [driveItem][] on OneDrive for Business
 - Security [alert][]
-- Teams [callRecord][] (preview)
+- Teams [callRecord][]
```
Comment (stephenjust) [inline concepts/webhooks.md:39]: Link definitions at the bottom of this page generally point to v1.0 for non-preview entities

Comment (GageAmes) [reply concepts/webhooks.md:39]: Thanks!  I added `?view=graph-rest-1.0` to the link definition in 7c7fc91

**Comment by GageAmes** [comment]
This will need to be updated to incorporate the changes from #8379 once it is merged.

**Comment by GageAmes** [comment]
The link to the beta documentation added in #8449 will also need to be updated here once #8449 is merged.

**Comment by stephenjust** [comment]
@GageAmes All of the dependent PRs have been merged.

**Comment by GageAmes** [comment]
@stephenjust I have ported over your changes from the other PRs.  Please have a look at the latest commits when you have a chance and let me know if I missed anything.

**Pull request merged**